### ISSUE TITLE
Invalid sass in _mixins.scss

### ISSUE THREAD

**Issue by andrewlock** [kelyvin/caffeine-theme#3]
When trying to build the project using `gulp build`, I get the following error: 

> 'Error: ".call-to-action-container .navigation .links li" failed to @extend "btn-cover".
       The selector "btn-cover" was not found.
       Use "@extend btn-cover !optional" if the extend should be able to fail.
        on line 47 of assets/scss/modules/_mixins.scss

This is caused by the mixin btn-cover-with-border: 

```
@mixin btn-cover-with-border {
    border: 1px solid $slate-white;

    &:hover {
        border: 1px solid $primary-color;
    }
    @extend btn-cover;
}
```

The final `@extend btn-cover;` should probably be changed to `@include btn-cover;`.

I can create a quick PR if you like though presumably the build is working for you, so maybe there is a separate issue?

**Comment by kelyvin** [comment]
You are absolutely correct and this should definitely be change to `@include`. 

I did not get this build issue because I had not updated my gulp-sass module version in a while. This most likely did not give me an issue because presumably there was a bug in my version of gulp-sass. I also experienced this issue on my other machine several weeks ago, so thanks for the catch! 

No need for a pull request, I will make the change.
